### ISSUE TITLE
New Device Request - H6022

### ISSUE THREAD

**Issue by q2doc** [bwp91/homebridge-govee#930]
* **Which model is this? Need the `Hxxxx` number.**
H6022

* **Which type of device is this? Light, heater, leak sensor, etc.**
Light

* **Which connection methods does this device have? Wifi? Bluetooth? LAN mode? All?**
All

* **Anything else?**
{"device":"21:0A:CF:C8:01:86:46:03","deviceName":"210ACFC801864603","model":"H6022","isLANDevice":true,"isLANOnly":true}

THX for supporting!

**Comment by SH1FT-W** [comment]
Yes, I agree and would also be happy if this could be supported with the next update!

**Comment by q2doc** [comment]
any news?

**Comment by stoyanland** [comment]
Do we expect any uptade?

**Comment by geruetzel** [comment]
I also would love this device to be supported.
